content-src/asrouter/templates/SimpleBelowSearchSnippet/_SimpleBelowSearchSnippet.scss

```diff
@@ -91,7 +91,12 @@
     }
 
     @media (max-width: 865px) {
-      margin: 0 60px 0 0;
+      margin: 0;
+      margin-inline-end: 60px;
```

Comment:
Do we really want the icon to be on the starting edge of the hover background? If so, you can simply do `margin-inline-start: 0` as the normal behavior otherwise would be to have 60px margin on the sides anyway.

![Screen Shot 2019-08-22 at 2 21 26 PM](https://user-images.githubusercontent.com/438537/63550820-4fb17f00-c4e8-11e9-972a-144b17cf1a5d.png)

However, not doing anything special for 865px could be fine too ??
![Screen Shot 2019-08-22 at 2 25 00 PM](https://user-images.githubusercontent.com/438537/63550994-c8b0d680-c4e8-11e9-96b6-855e69428186.png)

Reply:
Aaron did ask for the icon to be lined up with the search box, so the line-end margin is just to make space for the dismiss button on the other side.

Reply:
makes sense to change it to margin-inline-start, though